protocol/v2/ssv/validator/duty_executer.go

```diff
@@ -34,7 +33,9 @@ func (c *Committee) OnExecuteDuty(logger *zap.Logger, msg *types.EventMsg) error
 		return fmt.Errorf("failed to get execute committee duty data: %w", err)
 	}
 
-	logger = logger.With(fields.DutyID(fields.FormatCommitteeDutyID(c.Operator.Committee, c.BeaconNetwork.EstimatedEpochAtSlot(executeDutyData.Duty.Slot), executeDutyData.Duty.Slot)))
+	ids := types.OperatorIDsFromOperators(c.Operator.Committee)
+
+	logger = logger.With(fields.DutyID(fields.FormatCommitteeDutyID(ids, c.BeaconNetwork.EstimatedEpochAtSlot(executeDutyData.Duty.Slot), executeDutyData.Duty.Slot)))
```

Comment:
It would be great if we could separate the assertions here to improve code readability and maintainability.